lib/utils.js

```diff
@@ -79,12 +79,10 @@ module.exports = {
         }) : [];
     },
 
-    getTestCaseId (variables) {
-        const testCaseId = _.find(variables.members, {
-            key: 'rp.testCaseId'
-        });
+    getValueByKey (key, variables) {
```

Comment:
I recommend to rename the method to smth more specific because `getValueByKey` sounds like it just returns `object[key]`.